insights/parsers/iris.py

```diff
@@ -36,31 +38,47 @@ class IrisList(CommandParser, dict):
             product:      InterSystems IRIS
 
     Examples:
-        >>> iris_info['name']
-        'IRIS'
-        >>> iris_info['status']
+        >>> iris_info[0]['status']
         'running, since Tue Jun 27 01:55:25 2023'
         >>> iris_info.is_running
         True
+        >>> iris_info.default['status']
+        'running, since Tue Jun 27 01:55:25 2023'
     """
 
     def parse_content(self, content):
+        self.default = {}
+        default_instance = False
+        item_instance = {}
         for line in content:
             if not line.strip():
                 continue
             if line.strip().startswith('Configuration'):
-                self['name'] = line.split()[1].strip('\'"')
+                if item_instance:
+                    self.append(item_instance)
+                if default_instance:
+                    self.default = item_instance
+                instance_name = line.split()[1].strip('\'"')
+                item_instance = {"instance_name": instance_name}
+                if "(default)" in line:
+                    default_instance = True
+                else:
+                    default_instance = False
             elif ":" in line:
                 key, value = line.split(":", 1)
-                self[key.strip()] = value.strip()
+                item_instance[key.strip()] = value.strip()
+        if item_instance:
+            self.append(item_instance)
+        if default_instance:
+            self.default = item_instance
```

Comment:
```python
        self.default = {}
        item_instance = {}
        for line in content:
            if not line.strip():
                continue
            if line.strip().startswith('Configuration'):
                instance_name = line.split()[1].strip('\'"')
                item_instance = {"instance_name": instance_name}
                if "(default)" in line:
                    self.default = item_instance
                self.append(item_instance)
            elif ":" in line and item_instance:
                key, value = line.split(":", 1)
                item_instance[key.strip()] = value.strip()
```

Please check the above code, it would be equivalent to yours, but a bit simple.